### PR TITLE
Sing in page

### PR DESCRIPTION
**Pull request by profileEnumerable**

### CODE REVIEW

File: UI/Campus-UI/angular.json [hunk 1]
```diff
@@ -123,6 +125,10 @@
           }
         }
       }
-    }},
-  "defaultProject": "Campus-UI"
+    }
+  },
+  "defaultProject": "Campus-UI",
+  "cli": {
+    "analytics": "7115b5bd-1e51-4d45-b1d3-cedbec17b6db"
```
Comment (antonAce) [inline UI/Campus-UI/angular.json:132]: Do we really need CLI analytics for UI?

File: UI/Campus-UI/src/app/sing-in/sing-in.component.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  ngOnInit(): void {
+    this.form = new FormGroup({
+      email: new FormControl(null, [Validators.required, Validators.email]),
+      password: new FormControl(null, [Validators.required, Validators.minLength(6)])
+    });
+  }
+
+  submit() {
+    if (this.form.invalid) {
+      return;
```
Comment (antonAce) [inline UI/Campus-UI/src/app/sing-in/sing-in.component.ts:28]: This check is redundant, you can remove it

File: UI/Campus-UI/src/app/sing-in/sing-in.component.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+export class SingInComponent implements OnInit {
+
+  form: FormGroup;
+
+  // constructor(private auth: SignInService, private router: Router) {
+  // }
+
+  ngOnInit(): void {
+    this.form = new FormGroup({
+      email: new FormControl(null, [Validators.required, Validators.email]),
```
Comment (antonAce) [inline UI/Campus-UI/src/app/sing-in/sing-in.component.ts:21]: According to the API map, we use the 'login' value on the authentication endpoint, not 'email' one.

File: UI/Campus-UI/src/app/sing-in/sing-in.component.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+                  Password is required
+                </small>
+                <small *ngIf="form.get('password').errors.minlength">
+                  Password must contains {{form.get('password').errors.minlength.requiredLength}} characters,
+                  presently it is containing {{form.get('password').errors.minlength.actualLength}} symbols
+                </small>
+              </div>
+              <span class="forgot-pass"><a href="#">Forgot your password ?</a></span>
+            </div>
+            <button type="submit" class="sign-in__btn" [disabled]="form.invalid">Sign in</button>
```
Comment (antonAce) [inline UI/Campus-UI/src/app/sing-in/sing-in.component.html:44]: You can add 'touched' check like this: [disabled]="form.invalid || !form.touched"

Comment (antonAce) [inline UI/Campus-UI/src/app/sing-in/sing-in.component.html:44]: You can add 'touched' check like this: [disabled]="form.invalid || !form.touched"

File: UI/Campus-UI/angular.json (hunk 1 above)
Comment (profileEnumerable) [reply UI/Campus-UI/angular.json]: Removed

File: UI/Campus-UI/src/app/sing-in/sing-in.component.ts (hunk 2 above)
Comment (profileEnumerable) [reply UI/Campus-UI/src/app/sing-in/sing-in.component.ts]: Removed

File: UI/Campus-UI/src/app/sing-in/sing-in.component.ts (hunk 3 above)
Comment (profileEnumerable) [reply UI/Campus-UI/src/app/sing-in/sing-in.component.ts:21]: Unfortunately we had approved UI concept,and we'll spend valuable effort to change it

File: UI/Campus-UI/src/app/sing-in/sing-in.component.html (hunk 4 above)
Comment (profileEnumerable) [reply UI/Campus-UI/src/app/sing-in/sing-in.component.html]: Added

File: UI/Campus-UI/src/app/sing-in/sing-in.component.ts (hunk 3 above)
Comment (antonAce) [reply UI/Campus-UI/src/app/sing-in/sing-in.component.ts:21]: Promise me to fix it in the next sprints!

**Pull request merged**